nodes/put-object/put-object.js

```diff
@@ -108,6 +110,11 @@ module.exports = function (RED) {
         return;
       }
 
```

Comment:
ACL should be fetched from msg.acl if not specified in ui. That can be achieved with the following:

```js
if(!acl) {
   acl = msg.acl ? msg.acl : null;
}
```